source/neuropod/bindings/java/src/main/java/com/uber/neuropod/NeuropodTensor.java

```diff
@@ -87,7 +88,7 @@ public LongBuffer toLongBuffer() {
         if (isFromJava) {
             return buffer.asLongBuffer();
         }
-        LongBuffer ret = LongBuffer.allocate((int)getNumberOfElements()).put(buffer.asLongBuffer());
+        LongBuffer ret = LongBuffer.allocate((int) getNumberOfElements()).put(buffer.asLongBuffer());
```

Comment:
@VivekPanyam 

You can see some parts of to<Type>Buffer are not covered (only toFloatBuffer). 

This is because there is an Addition Test with model that has only Float Input and Output tensors.

Current code can be executed only if we have Output Tensor and call to<Type>Buffer(). We can create Output Buffer only using "infer" call. I don't want to add some hack that would allow us to create Output Tensors of different types only for test.

Instead, I will add test with Model that covers all different types (we use Addition model with all possible types to test our Go service with neuropod and I will use it as a base).

If you are ok with that, we can proceed with this PR and I will continue to work on test coverage.